app/components/offering-editor.js

```diff
@@ -0,0 +1,199 @@
+/* global moment */
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const { Component, computed, isEmpty, ObjectProxy, RSVP } = Ember;
+const { notEmpty } = computed;
+const { all, Promise } = RSVP;
+const { PromiseArray } = DS;
+
+export default Component.extend({
+  init() {
+    this._super(...arguments);
+
+    this.setProperties({
+      instructors: [],
+      instructorGroups: [],
+      learnerGroups: [],
+      startDate: moment().format('MM/DD/YYYY'),
+      endDate: moment().add(1, 'days').format('MM/DD/YYYY')
+    });
+  },
+
+  classNames: ['offering-editor'],
+
+  singleOffering: true,
+  isMultiDay: false,
+  filter: '',
+
+  startDate: null,
+  endDate: null,
+  room: null,
+
+  instructors: null,
+  instructorGroups: null,
+  learnerGroups: null,
+
+  filteredCohorts: computed('cohorts.[]', 'learnerGroups.[]', 'filter', function() {
+    let cohortProxy = ObjectProxy.extend({
+      selectedLearnerGroups: [],
+
+      hasAvailableLearnerGroups: notEmpty('filteredAvailableLearnerGroups'),
+
+      filter: '',
+
+      filteredAvailableLearnerGroups: computed('content.learnerGroups.[]', 'content.learnerGroups.@each.allDescendants.[]', 'selectedLearnerGroups.[]', 'filter', function() {
+        let defer = RSVP.defer();
+        let proxy = this;
+        let filter = proxy.get('filter');
+        let exp = new RegExp(filter, 'gi');
+
+        let activeGroupFilter = function(learnerGroup) {
+          let searchTerm = `${learnerGroup.get('title')}${learnerGroup.get('allParentsTitle')}`;
+
+          return (
+            learnerGroup.get('title') !== undefined &&
+            proxy.get('selectedLearnerGroups') &&
+            exp.test(searchTerm) &&
+            !proxy.get('selectedLearnerGroups').contains(learnerGroup)
+          );
+        };
+
+        this.get('content.topLevelLearnerGroups').then((cohortGroups) => {
+          let learnerGroups = [];
+          let promises = [];
+
+          cohortGroups.forEach((learnerGroup) => {
+            learnerGroups.pushObject(learnerGroup);
+
+            let promise = new Promise((resolve) => {
+              learnerGroup.get('allDescendants').then((descendants) => {
+                learnerGroups.pushObjects(descendants);
+                resolve();
+              });
+            });
+
+            promises.pushObject(promise);
+          });
+
+          all(promises).then(() => {
+            defer.resolve(learnerGroups.filter(activeGroupFilter).sortBy('sortTitle'));
+          });
+        });
+
+        return PromiseArray.create({
+          promise: defer.promise
+        });
+      }),
+    });
+
+    let cohorts = this.get('cohorts') ? this.get('cohorts') : [];
+
+    return cohorts.map((cohort) => {
+      let proxy = cohortProxy.create({
+        content: cohort,
+        selectedLearnerGroups: this.get('learnerGroups')
+      });
```

Comment:
Nevermind.  The filter is no longer an option on this component.  If we want to implement it it would need to be further down the tree in the cohort learner group component.  In that case just remove the filter property from line 27